Pyrado/pyrado/tasks/reward_functions.py

```diff
@@ -408,3 +408,20 @@ def __call__(self, state: np.ndarray, act: np.ndarray, remaining_steps: int = No
         self.last_x_pos = state[self._idx_x_pos]
 
         return float(fwd_vel_rew - ctrl_cost)
+
+
+class QCartPoleSwingUpCustomRewFcn(RewFcn):
+    """ Custom reward function for QCartPoleSwingUpSim. """
+
+    def __init__(self, factor=0.9):
+        """
+        Constructor
+        :param factor: weighting factor of rotation error to position error
+        """
+        self.factor = factor
+
+    def __call__(self, err_s: np.ndarray, err_a: np.ndarray, remaining_steps: int = None) -> float:
+        assert isinstance(err_s, np.ndarray) and isinstance(err_a, np.ndarray)
```

Comment:
Please work with two indipendent `if not isinstance(...): \n raise pyrado.TypErr(given=..., expected_match=...)`